reporter/reporter.go

```diff
@@ -236,6 +245,17 @@ func (t *reporter) reportDerivedMetrics(span tracer.RawSpan) {
 	replaceTag(tags, "application", appName, appFound)
 	replaceTag(tags, "service", serviceName, svcFound)
 
+	redMetricsCustomTags := t.application.Map()
+	if t.redMetricsCustomTagKeys != nil {
+		for _, key := range t.redMetricsCustomTagKeys {
+			if value, found := getAppTag(key, "", span.Tags); found {
+				tags[key] = value
+				redMetricsCustomTags[key] = value
+			}
+		}
+		t.heartbeater.Send(redMetricsCustomTags)
```

Comment:
not sure the context for this.  heartbeat no longer functions as a heartbeat, but more of reporter that reports metrics called heartbeat?